### PR TITLE
Add systemtest.mk file to include changes to enable testing on cygwin

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Resolves: https://github.com/AdoptOpenJDK/openjdk-systemtest/issues/199
Signed-off-by: Mesbah Alam <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Tested in this build: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/475/console

File: systemtest/playlist.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
-	
+	<include>systemtest.mk</include>
 	<test>
```
Comment (renfeiw) [inline systemtest/playlist.xml]: As you have defined a set of make variables (e.g., EXPORT, PERL...), do you want to use them in the test command?

File: systemtest/systemtest.mk [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+EMPTY=
+SPACE=$(EMPTY) $(EMPTY)
+
+define NEWLINE
+
+
+endef
```
Comment (llxia) [inline systemtest/systemtest.mk]: Where are `SPACE ` and `NEWLINE` used?

**Comment by llxia** [comment]
fyi @AdamBrousseau  after this PR, we should be able to turn on wins system test builds.

File: systemtest/playlist.xml (hunk 1 above)
Comment (renfeiw) [reply systemtest/playlist.xml]: There are three failures that are unrelated and excluded in the #708.

**Pull request merged**